### PR TITLE
Add custom wallet dropdown

### PR DESCRIPTION
**Pull request by muneeb-i-khan**

### CODE REVIEW

File: components/NavBar/index.tsx [hunk 1]
```diff
@@ -1,17 +1,68 @@
...
+        select(wallet.adapter.name);
+      } else {
+        select(PhantomWalletName);
+      }
 
-const WalletMultiButtonDynamic = dynamic(
-  async () =>
-    (await import("@solana/wallet-adapter-react-ui")).WalletMultiButton,
-  { ssr: false }
-);
```
Comment (pratikbuilds) [inline components/NavBar/index.tsx:12]: Why did you remove this code any reasons ?

File: components/NavBar/index.tsx [hunk 2]
```diff
@@ -30,10 +81,107 @@ function NavBar() {
...
+                      }}
+                    >
+                      Disconnect Wallet
+                    </Button>
+                  </Flex>
+                )}
+              </VStack>
+            </MenuItem>
+          </MenuList>
+        </Menu>
```
Comment (pratikbuilds) [inline components/NavBar/index.tsx]: Don't you think this logic can be it's own component rather than putting all this in the navbar

File: pages/_app.tsx [hunk 3]
```diff
@@ -4,22 +4,48 @@ import { theme } from "../theme/theme";
...
-  const endpoint = "https://api.devnet.solana.com";
+  const wallets = useMemo(
+    () => [
+      new PhantomWalletAdapter(),
+      new SolflareWalletAdapter(),
+      new GlowWalletAdapter(),
+      new TorusWalletAdapter(),
+    ],
+    []
+  );
```
Comment (pratikbuilds) [inline pages/_app.tsx]: This is being managed in WalletWrapper.tsx logic from

File: pages/_app.tsx [hunk 4]
```diff
@@ -4,22 +4,48 @@ import { theme } from "../theme/theme";
...
+              <Layout>
+                <Component {...pageProps} />
+              </Layout>
+            ) : (
+              <Text>Undefined</Text>
+            )}
+          </WalletProvider>
+        </ChakraProvider>
+      </ConnectionProvider>
     </div>
```
Comment (pratikbuilds) [inline pages/_app.tsx]: The logic is being repeated and i dont know why you have removed walletWrapper all this logic could be done there

File: components/NavBar/index.tsx (hunk 2 above)
Comment (muneeb-i-khan) [reply components/NavBar/index.tsx]: I will put all this logic inside a new component and then use that here. Will that be fine?

File: pages/_app.tsx (hunk 4 above)
Comment (muneeb-i-khan) [reply pages/_app.tsx]: I was getting some trouble doing it with Walletwrapper but I guess I have figured that out. I will try to use WalletWrapper rather than writing a whole new logic.

File: components/NavBar/index.tsx (hunk 2 above)
Comment (pratikbuilds) [reply components/NavBar/index.tsx]: Ya that would be fine

File: pages/_app.tsx (hunk 4 above)
Comment (muneeb-i-khan) [reply pages/_app.tsx]: > The logic is being repeated and i dont know why you have removed walletWrapper all this logic could be done there

**Comment by muneeb-i-khan** [comment]
@0xPratik I made the changes you asked. You can now have a look.
